axelrod/tests/unit/test_filters.py

```diff
@@ -16,6 +16,14 @@ class TestStrategy(object):
             'makes_use_of': ['game', 'length']
         }
 
+    class TestStrategy2(object):
+        classifier = {
+            'stochastic': True,
+            'inspects_source': False,
+            'memory_depth': 'infinity',
```

Comment:
nope. looking at the code, it's definitely there to cope with strategies having 'infinity' but I can't see any that do. Perhaps there were at some point.

We can either get rid of the line or add a test for it. Which do we prefer?